### PR TITLE
Resolve a Few Mullvad Issues

### PR DESCRIPTION
**Pull request by JeordyR**
Changes
* Added VPNSP to README
* Updated GetUser function to strip spaces, mullvad includes spaces on their site by default don't accept spaces through openvpn, so strip them for convenience if users copy/paste directly from site
* Corrected GetMullvadCountry param to use MullvadCountryChoices instead of MullvadCityChoices
* Updated Dockerfile to set COUNTRY to "sweden" by default instead of "Sweden" as the mappings in mullvad.go are all lower-case and the GetValueIfInside function from github.com/qdm12/golibs/params does not .ToLower in its comparison
  * May be worth updating that lib to .ToLower for its comparison, or have an IgnoreCase flag or something

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ ENV VPNSP=pia \
     PORT_FORWARDING=off \
     PORT_FORWARDING_STATUS_FILE="/forwarded_port" \
     # Mullvad only
-    COUNTRY=Sweden \
+    COUNTRY=sweden \
```
Comment (qdm12) [inline Dockerfile]: It should be case insensitive right? Or did I forget to code that :eyes:

File: Dockerfile [hunk 2]
```diff
@@ -79,4 +79,4 @@ RUN apk add -q --progress --no-cache --update openvpn ca-certificates iptables u
     adduser nonrootuser -D -H --uid 1000 && \
     chown nonrootuser -R /etc/unbound /etc/tinyproxy && \
     chmod 700 /etc/unbound /etc/tinyproxy
-COPY --from=builder --chown=1000:1000 /tmp/gobuild/entrypoint /entrypoint
\ No newline at end of file
+COPY --from=builder --chown=1000:1000 /tmp/gobuild/entrypoint /entrypointex
```
Comment (qdm12) [inline Dockerfile]: What's wrong with entrypoint? :smile:

File: internal/params/mullvad.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 // GetMullvadCountry obtains the country for the Mullvad server from the
 // environment variable COUNTRY
 func (p *paramsReader) GetMullvadCountry() (country models.MullvadCountry, err error) {
-	choices := append(constants.MullvadCityChoices(), "")
+	choices := append(constants.MullvadCountryChoices(), "")
```
Comment (qdm12) [inline internal/params/mullvad.go]: Thanks! :+1:

File: internal/params/openvpn.go [hunk 4]
```diff
@@ -21,6 +21,7 @@ func (p *paramsReader) GetUser() (s string, err error) {
 	} else if len(s) == 0 {
 		return s, fmt.Errorf("USER environment variable cannot be empty")
 	}
+	s = strings.Replace(s, ' ', '', -1)
```
Comment (qdm12) [inline internal/params/openvpn.go]: Why is this needed? You got white spaces where? Trailing spaces?

File: internal/params/mullvad.go (hunk 3 above)
Comment (JeordyR) [reply internal/params/mullvad.go]: 👍

File: Dockerfile (hunk 2 above)
Comment (JeordyR) [reply Dockerfile]: Woops, fix incoming :P

File: internal/params/openvpn.go (hunk 4 above)
Comment (JeordyR) [reply internal/params/openvpn.go]: Not 100% needed, but a nice to have if you want.

Mullvad includes spaces in the username they give you on their site by default but don't accept spaces connecting through openvpn, so stripping them for convenience if users copy/paste directly from site.

Comment (qdm12) [reply internal/params/openvpn.go]: Hmm I guess no one has a username with spaces, so good idea.

You can use `strings.ReplaceAll` instead though :wink`

Comment (qdm12) [reply internal/params/openvpn.go]: Also I think you forgot to import `"strings"` at the top so the thing does not compile.

As mentioned, you might want to try [the development container](https://github.com/qdm12/private-internet-access-docker#development) as it lints warnings/errors and auto-imports everything for you 👍

File: Dockerfile (hunk 1 above)
Comment (qdm12) [reply Dockerfile]: I just pushed a small commit updating golibs so it treats all environment variables without case sensitivity (lowercase everything).
